include/enumerations/medium.hpp

```diff
@@ -55,11 +55,11 @@ constexpr auto material_systems() {
   return material_systems;
 }
 
-constexpr auto element_types() {
-  constexpr int total_element_types = 8;
+constexpr auto medium_tags() {
+  constexpr int total_medium_tags = 8;
```

Comment:
Why did you change the nomenclature here?

Reply:
fixed.